### ISSUE TITLE
WAPharoWalkback can show method temporaries again

### ISSUE THREAD

**Issue by jbrichau** [SeasideSt/Seaside#1191]
The issue in https://github.com/seasidest/seaside/issues/689 no longer applies in Pharo 8.

We can re-enable the rendering of the method arguments and temporaries in the debugger walkback rendering.

**Comment by jbrichau** [comment]
Fixed in 3e430a7d294fdf3784abd62ec55fdfbedd20ffc8

**Comment by marschall** [comment]
Hello @jbrichau I noted `#expectedFailures` tests for

```st
SystemVersion current major >= 8
```

but `#tempNamesAndValuesIn:do:` tests for

```st
SystemVersion current major >= 9
```

Is this intentional or am I missing something?

**Comment by jbrichau** [comment]
@marschall woops!
Thx for noticing. That's a mistake. Fixed in 20e082861098f511fc2fea1bdb463de8560ec25f

Thx for watching ;)
